File: backend/spring-workspace/spring/src/main/java/bootProject/Hyperlink.java
package bootProject;

import java.util.Date;

public class Hyperlink {
	private String link;
	private String[] tags;
	private Date addedAt, lastEditedAt;
	
	public Hyperlink(String link, Date addedAt, String[] tags) {
		this.link = link;
		this.addedAt = addedAt;
		this.lastEditedAt = addedAt;
		this.tags = tags;
	}	

	public void setLink(String link) 			   { this.link = link; } 
	public void setTags(String[] tags) 			   { this.tags = tags; } 
	public void setLastEditedAt(Date lastEditedAt) { this.lastEditedAt = lastEditedAt; }
	
	public String getLink() 		 { return link; }
	public String[] getTags() 		 { return tags; }
	public Date getLastEditedAt() 	 { return lastEditedAt; }
	public Date getAddedAt() 	 	 { return addedAt; }
	
}

File: sampleclient/model/hyperlink.py
class Hyperlink(object):
	
	def __init__(self):
		self.link = ""
		self.lists = []
		self.comments = []

	def __init__(self, link, lists, comments):
		self.link = link
		self.lists = lists
		self.comments = comments

	def __str__(self):
		return "Hyperlink: %s" %link